### PR TITLE
Minor README tweaks to help newbies

### PR DESCRIPTION
**Pull request by duglin**
Signed-off-by: Doug Davis <dug@us.ibm.com>

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -6,13 +6,14 @@
 
 ```bash
 git clone https://github.com/opencontainers/runc
```
Comment (pnasrat) [inline README.md]: Probably also want to mention cloning into  $GOPATH/opencontainers else you'll get build errors.

Comment (pnasrat) [inline README.md]: Actually #14 has some discussion on that ignore.

Comment (duglin) [inline README.md]: yea, I thought about modifying that part too in this PR but I didn't want to overlap with #14.
I can do it if people want though....

Comment (stevvooe) [inline README.md]: @duglin Let's fix this here. Use `go get -d`. We can close #14.

Comment (duglin) [inline README.md]: ok done - see if I got it right - wasn't sure what to do about the 'cd' step

**Comment by mrunalp** [comment]
@duglin Please rebase :)

**Comment by duglin** [comment]
done

**Comment by duglin** [comment]
@mrunalp rebased

**Comment by duglin** [comment]
Per discussion at Docker's DGAB, reverted back to `git clone` instead of `go get`

**Comment by duglin** [comment]
ping @crosbymichael

**Comment by mrunalp** [comment]
LGTM

**Comment by crosbymichael** [comment]
LGTM

**Pull request merged**